docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
 and cluster role bindings from clusters running a SAP payload.
 
-**Conditional data**: This data is collected only if the "installers.datahub.sap.com" resource is found in the cluster.
+### API Reference
+- https://pkg.go.dev/github.com/openshift/client-go/authorization/clientset/versioned/typed/authorization/v1
+- https://pkg.go.dev/github.com/openshift/client-go/security/clientset/versioned/typed/security/v1
 
-Relevant OpenShift API docs:
-  - https://pkg.go.dev/github.com/openshift/client-go/authorization/clientset/versioned/typed/authorization/v1
-  - https://pkg.go.dev/github.com/openshift/client-go/security/clientset/versioned/typed/security/v1
+### Sample data
+- [docs/insights-archive-sample/config/securitycontextconstraint](./insights-archive-sample/config/securitycontextconstraint)
+- [docs/insights-archive-sample/config/clusterrolebinding](./insights-archive-sample/config/clusterrolebinding)
 
-* Location in archive: config/securitycontentconstraint/, config/clusterrolebinding/
-* Id in config: clusterconfig/sap_config
-* Since versions:
-  - 4.6.20+
-  - 4.7+
+### Location in archive
+- `config/{securitycontentconstraint|clusterrolebinding}/{name}.json`
+
+### Config ID
+`clusterconfig/sap_config`
+
+### Released version
+- 4.7.0
+
+### Backported versions
+- 4.6.20+
+
+### Changes
+None
 
 
 ## SAPDatahubs
 
-collects `datahubs.installers.datahub.sap.com` resources from SAP/SDI clusters.
+Collects `datahubs.installers.datahub.sap.com`
+resources from SAP/SDI clusters.
+
+### API Reference
+None
 
-* Location in archive: customresources/installers.datahub.sap.com/datahubs/{namespace}/{name}.json
-* Id in config: clusterconfig/sap_datahubs
-* Since versions:
-  - 4.7.5+
-  - 4.8+
+### Sample data
+- [docs/insights-archive-sample/customresources/installers.datahub.sap.com/datahubs/sdi/default.json](./insights-archive-sample/customresources/installers.datahub.sap.com/datahubs/sdi/default.json)
+
+### Location in archive
+- `customresources/installers.datahub.sap.com/datahubs/{namespace}/{name}.json`
+
+### Config ID
+`clusterconfig/sap_datahubs`
+
+### Released version
+- 4.8.2
+
+### Backported versions
+- 4.7.5+
+- 4.6.26+
+
+### Changes
+None
 
 
 ## SAPPods
 
-collects information about pods running in SAP/SDI namespaces.
-Only pods with a failing status are collected.
-Failed pods belonging to a job that has later succeeded are ignored.
+Collects information about pods running in SAP/SDI namespaces.
+
+- Only pods with a failing status are collected.
+- Failed pods belonging to a job that has later succeeded are ignored.
+
+### API Reference
+- https://pkg.go.dev/k8s.io/client-go/kubernetes/typed/core/v1
+- https://pkg.go.dev/k8s.io/client-go/kubernetes/typed/batch/v1
+- https://pkg.go.dev/k8s.io/client-go/dynamic
+
+### Sample data
+None
+
+### Location in archive
+- `config/pod/{namespace}/{name}.json`
 
-**Conditional data**: This data is collected only if the "installers.datahub.sap.com" resource is found in the cluster.
```

Comment:
I thought we eventually agreed on keeping this line. It is important because "the gatherer completed successfully" and "there are no data in the archive" **does not imply** "there were no resources on the cluster".